stream_chat/async_chat/client.py

```diff
@@ -566,10 +567,14 @@ async def export_channel(
                     "messages_since": messages_since,
                     "messages_until": messages_until,
                 }
-            ]
+            ],
+            **options,
         )
 
-    async def export_channels(self, channels: Iterable[Dict]) -> StreamResponse:
+    async def export_channels(
+        self, channels: Iterable[Dict], **options: Any
+    ) -> StreamResponse:
+        options = options or {}
```

Comment:
`options` will always be a dict here:
<img width="540" alt="Screenshot 2022-01-25 at 11 25 03" src="https://user-images.githubusercontent.com/40196/150959223-188a4551-a7bd-417a-98cb-0e85a31787d4.png">